simulation/__main__.py

```diff
@@ -65,12 +68,28 @@ def restrict_between_0_and_1(input_value: float):
         help="A factor by which current (2021) air source heat pump unit+install costs will have declined by, as of the end of 2022",
     )
 
+    def is_isoformat_datetime_string(string):
+        datetime.datetime.fromisoformat(string)
+        return string
+
+    parser.add_argument(
+        "--seed",
+        default=datetime.datetime.now().isoformat(),
+        type=is_isoformat_datetime_string,
+        help="""
+        Seed for random number generator. Default is now.
+    """,
```

Comment:
This looks like it has formatted incorrectly, will it not fit on one line with `"` ?
```suggestion
        help="Seed for random number generator. Default is now.",
```

Reply:
No, it's how `black` does it.